src/snowflake/cli/_plugins/workspace/commands.py

```diff
@@ -162,3 +165,30 @@ def deploy(
         validate=validate,
     )
     return MessageResult("Deployed successfully.")
+
+
+@ws.command(requires_connection=True)
+@with_project_definition()
+def drop(
+    entity_id: str = typer.Option(
+        help=f"""The ID of the entity you want to drop.""",
+    ),
+    # TODO The following options should be generated automatically, depending on the specified entity type
```

Comment:
Suggestion: create a ticket for this work and attach the ticket # here

Reply:
In general I like the idea of attaching ticket numbers to TODO comments, but I didn't really see it's used in our codebase. Is this a pattern we want to introduce?

Reply:
It's OK, just as long as we track it.